fischerlol/mod-level-15-boost
# fix. build

## Changes Proposed:
- Change the way you write commands.
- Correct some tabs and line breaks
- Other minor issues

## Issues Addressed:

- Closes

## SOURCE:


## Tests Performed:

- Windows 10

## How to Test the Changes:


1.I made these changes to see if it passes the tests, although I still need to make the changes within the game as well. I didn't try it in the game, because I'm not sure what the module is good for. I looked at it above, and made some corrections, to see if it passes the tests in Linux here with the github actions that are defined, linked to the AzerothCore repository.
